runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -304,13 +301,33 @@ else if (paused && !skillPaused)
 			pauseSkill.setText("Pause");
 		}
 
+		// Update information labels
 		// Update exp per hour separately, every time (not only when there's an update)
-		expHour.setText(htmlLabel("XP/Hour: ", xpSnapshotSingle.getXpPerHour()));
+		topLeftStat.setText(htmlLabel(xpTrackerConfig.xpPanelLabel1(), xpSnapshotSingle));
+		topRightStat.setText(htmlLabel(xpTrackerConfig.xpPanelLabel2(), xpSnapshotSingle));
+		bottomLeftStat.setText(htmlLabel(xpTrackerConfig.xpPanelLabel3(), xpSnapshotSingle));
+		bottomRightStat.setText(htmlLabel(xpTrackerConfig.xpPanelLabel4(), xpSnapshotSingle));
+	}
+
+	static String htmlLabel(XpPanelLabel panelLabel, XpSnapshotSingle xpSnapshotSingle)
+	{
+		String key = panelLabel.getActionKey(xpSnapshotSingle) + ": ";
+		if (panelLabel == XpPanelLabel.TIME_TO_LEVEL)
```

Comment:
Why is this not using the value func like rest?

Reply:
10-4. TTL was the edge case that used a `String` and `valueFunc` expects an `int`.

I'll change `XpPanelLabels`' `valueFunc` to return an already formatted `String` to keep things consistent.